flow/activities/maintenance_activity.go

```diff
@@ -51,8 +52,10 @@ func (a *MaintenanceActivity) GetAllMirrors(ctx context.Context) (*protos.Mainte
 	maintenanceMirrorItems, err := pgx.CollectRows(rows, func(row pgx.CollectableRow) (*protos.MaintenanceMirror, error) {
 		var info protos.MaintenanceMirror
 		var createdAt time.Time
-		err := row.Scan(&info.MirrorId, &info.MirrorName, &info.WorkflowId, &createdAt, &info.IsCdc)
+		var updatedAt time.Time
+		err := row.Scan(&info.MirrorId, &info.MirrorName, &info.WorkflowId, &createdAt, &updatedAt, &info.IsCdc)
```

Comment:
```suggestion
		err := row.Scan(&info.MirrorId, &info.MirrorName, &info.WorkflowId, &info.MirrorCreatedAt, &info.MirrorUpdatedAt, &info.IsCdc)
```

Reply:
I'm not sure if it can be directly put to proto

Reply:
It can, already does for a few fields

Reply:
```
can't scan into dest[3] (col: created_at): cannot scan timestamp (OID 1114) in binary format into **timestamppb.Timestamp
```

Reply:
oh, my eyes glazed over `timestamppb.New`